R/calcFEdemand.R

```diff
@@ -581,11 +581,11 @@ calcFEdemand <- function(subtype = "FE") {
 
     datatmp <- data
     # Take the names of reminditems without scenario and rcp dimension
-    names_NoScen <- lapply(getNames(reminditems), function(name) {
+    names_NoScen <- unique(lapply(getNames(reminditems), function(name) {
       strsplit(name, ".", fixed = TRUE)[[1]] %>%
         `[`(!tail(getSets(reminditems), -2) %in% c("scenario", "rcp")) %>%
         paste(collapse = ".")
-    })
+    }))
```

Comment:
Why not
```
    names_NoScen <- lapply(unique(getNames(reminditems)), function(name) {
      strsplit(name, ".", fixed = TRUE)[[1]] %>%
        `[`(!tail(getSets(reminditems), -2) %in% c("scenario", "rcp")) %>%
        paste(collapse = ".")
    })
```
Seems more intuitive to me …

Reply:
Because this way, the duplicates remain. The function within lapply removes ssp and rcp from the 3rd madrat dimension and leaves the carriers. This list of carriers has the duplicates. We only need unique carriers.

Reply:
kk